src/ApplicationManager.cpp

```
@@ -856,6 +856,23 @@ void ApplicationManager::handleSpoolmanSynced(const AppMessage& msg) {
     }
 
 #ifndef NATIVE_TEST
+    // Re-evaluate LED state using Spoolman weight — catches tags without weight data
+    // (OpenSpool, GenericUID, TigerTag) where Spoolman knows the remaining weight
+    if (msg.payload.spoolmanSynced.success && kgRemaining > 0.0f &&
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)